src/components/Tabs/index.js

```diff
@@ -1 +0,0 @@
-export * from './Tabs';
```

Comment:
Why did you delete that file? Bring it back, it allows you to shorten component import statements.